lib/npm-check-updates.js

```diff
@@ -97,20 +99,54 @@ function createDependencyTable() {
             middle: ''
         }
     });
+    return baseTable;
+}
+
+/**
+ * @param {object} args
+ * @param {object} options
+ * @returns {Promise<object>}
+ */
+async function getOwnerPerDependency(args, options) {
+    const packageManager = vm.getPackageManager(options.packageManager);
+    args.owner = {};
+    await Promise.all(
+        Object.keys(args.to).map(async (dep) => {
+            const from = args.from[dep] || null;
+            const to = args.to[dep] || null;
+            const ownerChanged = await packageManager.packageAuthorChanged(dep, from, to);
+            // eslint-disable-next-line require-atomic-updates
+            args.owner[dep] = ownerChanged;
+        })
+    );
+    return args;
 }
 
 /**
  * @param {object} args
  * @param {object} args.from
  * @param {object} args.to
- * @returns {Table}
+ * @param {object} args.owner
+ * @param {object} options
+ * @returns {Promise<Table>}
  */
-function toDependencyTable(args) {
+async function toDependencyTable(args, options) {
     const table = createDependencyTable();
-    const rows = Object.keys(args.to).map(dep => {
-        const from = args.from[dep] || '';
-        const to = versionUtil.colorizeDiff(args.from[dep], args.to[dep] || '');
-        return [dep, from, '→', to];
+    let mappedArgs = [];
+    if (options.ownerChanged) {
+        const newArgs = await getOwnerPerDependency(args, options);
+        mappedArgs = _.cloneDeep(newArgs);
+    } else {
+        mappedArgs = _.cloneDeep(args);
+    }
+    const rows = Object.keys(mappedArgs.to).map(dep => {
+        const from = mappedArgs.from[dep] || '';
+        const to = versionUtil.colorizeDiff(mappedArgs.from[dep], mappedArgs.to[dep] || '');
+        let owner = '';
+        if (options.ownerChanged) {
+            owner = mappedArgs.owner && mappedArgs.owner[dep] ? '*owner changed*' : '';
```

Comment:
How could that be displayed on the table? 🤔

Reply:
I somewhat prefer `*owner changed*` as the 

Listing the actual changed owners would make sense in `--loglevel verbose` if we were to expand results beyond a single line.

Reply:
Agreed, that might make the whole thing noisy. Maybe additionally a "some owners changed while checking for updates, execute with --loglevel verbose to see the full list" at the end of the listing in case something changed?

Reply:
On the other hand, since currently getting the owner changed information is already opt-in, wouldn't that be enough intent to actually get the owner information as well?

Reply:
What about: `New owner: OWNER_NAME`? That generally still fit on one line.

Reply:
The current owners (and hence the new owners) are pretty easy to look up on npmjs.com tho, whereas it's harder to get information about past owners. Also there can be multiple owners. Maybe "No longer owner(s): []"?

Reply:
Yes, but is looking up past owners useful? The security risk is that the new owner is untrustworthy.

Reply:
That's true. Well, personally I'd be fine if it just would print additional lines, one for old and one for new owner(s), so all information is available if you request it using `-o` anyway. But having just the new owner(s) in the same line would be equally fine for me. Might make even sense in the light of https://github.com/tjunnone/npm-check-updates/issues/570#issuecomment-573380990, if potentially additional information gets added like license change or changelog.

Reply:
Another idea (thanks for the discussion) is `old → new` with `new` colored red.

e.g. (fabricated)

```
 htmlparser2              ^3.10.1  →   ^4.0.0  feedic → foo
 moment                   ^2.22.2  →  ^2.24.0
 react-click-n-hold        ^1.0.6  →   ^1.0.7
```

Reply:
Another option that I think would be reasonable would be to add something like `--full` which would be expected to break the single-line formatting and would allow room for additional information to be added in the future, as suggested.

Reply:
That could get interesting if multiple owners get added and removed (long line). Also there might be a problem with accessibility (color blind) if owners are only added or removed (no arrow, or arrow pointing to / originating from nothing).

But

```
htmlparser2              ^3.10.1  →   ^4.0.0

                       Owner added:  foo, bar, baz
                           removed:  baz, bar, foo

              License changed from:  MIT
                                to:  GPL

moment                   ^2.22.2  →  ^2.24.0
react-click-n-hold        ^1.0.6  →   ^1.0.7
```

isn't exactly pretty :grimacing:

Reply:
> Another option that I think would be reasonable would be to add something like --full

So, leaving `*owner changed*` as is and showing more information with the `--full` flag?

Reply:
> isn't exactly pretty 😬

That's not so bad for the full view.

> So, leaving *owner changed* as is and showing more information with the --full flag?

That's what I'm leaning towards.

Reply:
@stoically I'm going to get this PR merged as-is, but I'm open to enhanced functionality in a separate PR!